### ISSUE TITLE
Maintenance status

### ISSUE THREAD

**Issue by vkammerer** [goderbauer/contact_picker#24]
Hello @goderbauer, do you still want to maintain this plugin?

I am asking this because I see that you haven't been replying to issues nor commenting on PRs opened in this repo since may 2018.

In such situations, I would usually create a plugin myself with a variation of the name and include upgrades / bug fixes, but I realize that others have already done this:
- https://pub.dev/packages/native_contact_picker
- https://pub.dev/packages/easy_contact_picker
- https://pub.dev/packages/native_contact_dialog

So I would rather avoid squatting the "yet_another_contact_picker" package name, which would add even more confusion to that which one has to go through for such a simple functionality.

I would more specifically be interested in upgrading it to the latest Android embedding v2, but I guess there are other improvements that this plugin could benefit from.

Any reply would be appreciated :) thanks

**Comment by DRSchlaubi** [comment]
The new package does not seem to be very maintained as well

**Comment by jerem17** [comment]
I would also be interested in a new package. Everything I find doesn't work on Android, but works fine on iOS :/

**Comment by DRSchlaubi** [comment]
I made one myself you can find it at https://github.com/DRSchlaubi/contact_picker

**Comment by jerem17** [comment]
Thank you for your response! 
It allows you to retrieve email, phone etc?

**Comment by DRSchlaubi** [comment]
currently it allows you to retrieve a phone number or an email I'll probably add support for the whole contact at some point

**Comment by jerem17** [comment]
Oh, right, I thought phone and email were bundled together. Thanks for your help, it's already that ;)

**Comment by vkammerer** [comment]
@DRSchlaubi @jerem17 you may want to move that discussion to another thread.
This issue was opened as a reference for everyone considering using this plugin to understand what @goderbauer 's position is regarding its maintenance.

**Comment by Jayvd** [comment]
@DRSchlaubi @goderbauer 

I wanted to switch to other library, so I removed this library from `pubspec.yaml`

But after that, I cannot rebuild and run my app anymore..

I get this error instead. Can anyone tell me why is this happening? Please

`Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:45:24: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
    - (void)contactPicker:(CNContactPickerViewController *)picker
                           ^
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:45:1: note: annotate 'contactPicker:didSelectContactProperty:' with an availability attribute to silence this warning
    - (void)contactPicker:(CNContactPickerViewController *)picker
    ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:46:31: warning: 'CNContactProperty' is only available on iOS 9.0 or newer [-Wunguarded-availability]
        didSelectContactProperty:(CNContactProperty *)contactProperty {
                                  ^
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    In module 'Contacts' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Contacts.framework/Headers/CNContactProperty.h:21:12: note: 'CNContactProperty' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactProperty : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:45:1: note: annotate 'contactPicker:didSelectContactProperty:' with an availability attribute to silence this warning
    - (void)contactPicker:(CNContactPickerViewController *)picker
    ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:58:33: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
    - (void)contactPickerDidCancel:(CNContactPickerViewController *)picker {
                                    ^
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:58:1: note: annotate 'contactPickerDidCancel:' with an availability attribute to silence this warning
    - (void)contactPickerDidCancel:(CNContactPickerViewController *)picker {
    ^
                                                                            API_AVAILABLE(ios(9.0))
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:33:5: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:33:5: note: enclose 'CNContactPickerViewController' in an @available check to silence this warning
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:33:54: warning: 'CNContactPickerViewController' is only available on iOS 9.0 or newer [-Wunguarded-availability]
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
                                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:25:12: note: 'CNContactPickerViewController' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactPickerViewController : UIViewController
               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:33:54: note: enclose 'CNContactPickerViewController' in an @available check to silence this warning
        CNContactPickerViewController *contactPicker = [[CNContactPickerViewController alloc] init];
                                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:35:46: warning: 'CNContactPhoneNumbersKey' is only available on iOS 9.0 or newer [-Wunguarded-availability]
        contactPicker.displayedPropertyKeys = @[ CNContactPhoneNumbersKey ];
                                                 ^~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    In module 'Contacts' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Contacts.framework/Headers/CNContact.h:160:34: note: 'CNContactPhoneNumbersKey' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    CONTACTS_EXTERN NSString * const CNContactPhoneNumbersKey                    NS_AVAILABLE(10_11, 9_0);
                                     ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:35:46: note: enclose 'CNContactPhoneNumbersKey' in an @available check to silence this warning
        contactPicker.displayedPropertyKeys = @[ CNContactPhoneNumbersKey ];
                                                 ^~~~~~~~~~~~~~~~~~~~~~~~
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:47:25: warning: 'CNContactFormatter' is only available on iOS 9.0 or newer [-Wunguarded-availability]
      NSString *fullName = [CNContactFormatter stringFromContact:contactProperty.contact
                            ^~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    In module 'Contacts' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Contacts.framework/Headers/CNContactFormatter.h:42:12: note: 'CNContactFormatter' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNContactFormatter : NSFormatter <NSSecureCoding>
               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:47:25: note: enclose 'CNContactFormatter' in an @available check to silence this warning
      NSString *fullName = [CNContactFormatter stringFromContact:contactProperty.contact
                            ^~~~~~~~~~~~~~~~~~
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:48:62: warning: 'CNContactFormatterStyleFullName' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                                                           style:CNContactFormatterStyleFullName];
                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    In module 'Contacts' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Contacts.framework/Headers/CNContactFormatter.h:17:28: note: 'CNContactFormatterStyle' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    typedef NS_ENUM(NSInteger, CNContactFormatterStyle)
                               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:48:62: note: enclose 'CNContactFormatterStyleFullName' in an @available check to silence this warning
                                                           style:CNContactFormatterStyleFullName];
                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:51:37: warning: 'CNLabeledValue' is only available on iOS 9.0 or newer [-Wunguarded-availability]
                                       [CNLabeledValue localizedStringForLabel:contactProperty.label],
                                        ^~~~~~~~~~~~~~
    In module 'ContactsUI' imported from /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:6:
    In module 'Contacts' imported from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/ContactsUI.framework/Headers/CNContactPickerViewController.h:9:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/Contacts.framework/Headers/CNLabeledValue.h:18:12: note: 'CNLabeledValue' has been marked as being introduced in iOS 9.0 here, but the deployment target is iOS 8.0.0
    @interface CNLabeledValue<ValueType : id <NSCopying, NSSecureCoding>> : NSObject <NSCopying, NSSecureCoding>
               ^
    /Users/mariozagalo/.pub-cache/hosted/pub.dartlang.org/contact_picker-0.0.2/ios/Classes/ContactPickerPlugin.m:51:37: note: enclose 'CNLabeledValue' in an @available check to silence this warning
                                       [CNLabeledValue localizedStringForLabel:contactProperty.label],
                                        ^~~~~~~~~~~~~~
    9 warnings generated.
    diff: /Users/mariozagalo/FlutterProjects/MyTaskApp/ios/Pods/Manifest.lock: No such file or directory
    error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'contact_picker' from project 'Pods')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99. (in target 'Flutter' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone X.
`

**Comment by DRSchlaubi** [comment]
You might want to create this issue on the repo of the repo actually causing the issue

**Comment by vkammerer** [comment]
@Jayvd could you please delete your comment, which is unrelated to this issue?

**Comment by KapilPremware** [comment]
@Jayvd did you find any solution?
